.github/workflows/docker-connector-base-image-tests.yml

```
@@ -3,19 +3,19 @@
 
 # NOTE: This workflow does not publish the images to Docker Hub. It only builds them and tests them.
 
-name: Docker Image Tests
+name: Dockerfile Tests
```

```suggestion
name: Container Tests
```
for consistency of naming